.github/DEVELOPMENT.md

```diff
@@ -5,43 +5,106 @@ If you are having trouble, don't be afraid to [ask for help](./CONTRIBUTING.md#
 
 ## Prerequisites
 
-- [Install Node.js 14 LTS or newer with npm@7+](https://nodejs.org/en/download).
-  - If you're new to installing Node, a tool like [nvm](https://github.com/nvm-sh/nvm#install-script) can help you manage multiple version installations.
-- You will need [Google Chrome](https://www.google.com/chrome) to run browser-based tests locally.
+- If you're fixing an issue, make sure it's labeled with [`status: accepting prs`](https://github.com/mochajs/mocha/issues?q=is%3Aissue+is%3Aopen+label%3A%22status%3A+accepting+prs%22)
+- [Install the current Node.js LTS](https://nodejs.org/en/download)
+- [Install Google Chrome](https://www.google.com/chrome) if you want to run browser-based tests locally
 
 ## Setup
 
 1. Follow [Github's documentation](https://help.github.com/articles/fork-a-repo) on setting up Git, forking, and cloning.
 1. Execute `npm install` to install the development dependencies.
    - Do not use `yarn install` or `pnpm install`.
-   - Some optional dependencies may fail; you can safely ignore these unless you are trying to build the documentation.
-   - If you're sick of seeing the failures, run `npm install --ignore-scripts`.
-
-## Developing Mocha
-
-When you contribute to Mocha, you will probably want to try to run your changes on the test suite of another project. You can (and should) run the test suite of Mocha itself before committing, but also confirming that your changes give the expected result on another project.
-
-For example, [WebSocket.io](https://github.com/LearnBoost/websocket.io/):
-
-    $ git clone https://github.com/LearnBoost/websocket.io.git
-
-Retrieve websocket.io's dependencies, which will include the stable version of Mocha:
-
-    $ cd websocket.io/
-    $ npm install
-
-Replace the Mocha dependency by the current git repository:
-
-    $ cd node_modules/
-    $ mv mocha/ mocha.save
-    $ git clone https://github.com/mochajs/mocha.git
-
-Install Mocha's dependencies for the development version:
-
-    $ cd mocha
-    $ npm install
-
-Run websocket.io's test suite using the development version you just installed:
-
-    $ cd ../..
-    $ ./node_modules/.bin/mocha
+1. Create a new branch in your working copy.
+
+## Reproducing issues
+
+We recommend creating a folder named `sandbox` within this repo, and then using a child project within `sandbox` for each issue you want to repro.
```

Comment:
I actually have a different repro flow. I keep a folder at `~/repos/repros` as a sibling to `~/repos/mocha`. It's a Git repo with a branch for each test case / reproduction. The publicly interesting ones get pushed to https://github.com/JoshuaKGoldberg/repros.

Alternately, sometimes I use `~/repos/mocha-examples` to get starting reproductions. It's pretty handy.

I don't think we should recommend such a specific usage flow. Not every user is going to want this.

Proposal: how about making this more general, like saying having _some_ folder _somewhere_?